packages/core/src/BinaryDataManager/index.ts

```diff
@@ -1,11 +1,12 @@
-import concatStream from 'concat-stream';
 import { readFile, stat } from 'fs/promises';
 import type { BinaryMetadata, IBinaryData, INodeExecutionData } from 'n8n-workflow';
 import prettyBytes from 'pretty-bytes';
 import type { Readable } from 'stream';
 import { BINARY_ENCODING } from '../Constants';
 import type { IBinaryDataConfig, IBinaryDataManager } from '../Interfaces';
 import { BinaryDataFileSystem } from './FileSystem';
+// eslint-disable-next-line import/no-cycle
```

Comment:
cyclic dependencies have a whole set of issues, and that's why I've been trying to get rid of them. Can we please move this function to a separate file instead for now?

Reply:
Looking forward to breaking up n8n-workflow